api/logic/task_logic.go

```diff
@@ -33,12 +37,17 @@ func (this *L0TaskLogic) ListTasks() ([]*models.TaskSummary, error) {
 }
 
 func (this *L0TaskLogic) GetTask(taskID string) (*models.Task, error) {
```

Comment:
Possibly out of the scope of this PR, but we should be moving away from `this` when we can.

Reply:
Agreed, and also think it's out of scope.